docs/reference/requirement-sets/word-api-1-3-requirement-set.md

```diff
@@ -0,0 +1,316 @@
+---
+title: Word JavaScript API requirement set 1.3
+description: 'Details about the WordApi 1.3 requirement set'
+ms.date: 07/17/2019
+ms.prod: word
+localization_priority: Normal
+---
+
+# What's new in Word JavaScript API 1.3
+
+WordApi added more support for content controls, custom XML, and document-level settings.
+
+## API List
+
+The following is a complete list of APIs added as part of the WordApi 1.3 requirement set.
+
+| Class | Fields | Description |
```

Comment:
There's a problem with the formatting of this table that's resulting in it not rendering correctly.  

https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/word-api-1-3-requirement-set?branch=AlexJ-WorReqSets

![image](https://user-images.githubusercontent.com/3019529/61478888-7c053900-a947-11e9-8010-7dcfdaa6d1bf.png)